src/NServiceBus.Core/Features/FeaturesReport.cs

```diff
@@ -2,13 +2,10 @@ namespace NServiceBus.Features
 {
     using System.Collections.Generic;
 
-    class FeaturesReport
+    class FeaturesReport : List<FeatureDiagnosticData>
```

Comment:
Why keep this type around instead of returning  `List<FeatureDiagnosticData>` from `SetupFeatures()`?

Reply:
I thought that the FeaturesReport was added to the settings and therefore we would need a type for better accessibility, but that's not the case, removed it with an array.